docs/credential-helpers.md

```diff
@@ -45,7 +45,7 @@ volumes:
   helper: {}
 ```
 
-and for `.docker/config.yml`:
+and for `<PATH_TO_HOME_DIR>/.docker/config.json`:
 ```yaml
```

Comment:
The syntax identifier inside the codeblock, needs to be changed to json as well!

Reply:
@zoispag A good point. It's changed in the branch. I also removed the comments from the JSON codeblock, because comments are not allowed in JSON. They are shown as red errors in the resulting HTML.